README.md

```diff
@@ -26,6 +26,14 @@
 * contains(self, token) - Returns True if token is present in the trie, False if not.
 
 
```

Comment:
Also need to include information on your Insertion sort. For all sorting methods, make sure to include details on whether it is an "in-place" sort, or if it creates and returns a new object. Also include information on whether it is a stable or unstable sort.